### PR TITLE
Update screenshots for the SLO List widget docs

### PR DESCRIPTION
**Pull request by kaixintai**
### What does this PR do?

We recently made a UI change to the SLO List widget and the screenshots on the SLO List widget page are now outdated. This PR updates the copy and screenshots.
 
### Motivation

https://docs-staging.datadoghq.com/kai/slo-list-screenshots/dashboards/widgets/slo_list/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/dashboards/widgets/slo_list.md [hunk 1]
```diff
@@ -12,19 +12,19 @@ further_reading:
   text: "Building Dashboards using JSON" 
 ---
 
-The SLO List widget displays a subset of [SLOs][1], based on your query. 
+The SLO List widget displays a subset of [SLOs][1] over their primary time window. All other configured time windows are still available on the SLO's side panel.
```
Comment (buraizu) [inline content/en/dashboards/widgets/slo_list.md:15]:
```suggestion
The SLO List widget displays a subset of [SLOs][1] over their primary time window. All other configured time windows are available in the SLO's side panel on the SLO page. 
```

**Comment by estherk15** [comment]
@kaixintai can you keep the old images in the folders? This will impact translated docs that are looking for the old images.

**Review by estherk15 — CHANGES_REQUESTED** [review]
@kaixintai can you keep the old images in the folders? Removing them will impact the translated docs that still reference the old images.  🙇‍♀️

**Review by buraizu — APPROVED** [review]
Thanks, this LGTM overall but I left some optional minor suggestions to one sentence.

**Review by estherk15 — APPROVED** [review]

**Pull request merged**